README.md

```diff
@@ -1 +1,62 @@
-# module-ballerina-udp
\ No newline at end of file
+Ballerina UDP Library
+===================
+
+  [![Build](https://github.com/ballerina-platform/module-ballerina-udp/workflows/Build/badge.svg)](https://github.com/ballerina-platform/module-ballerina-udp/actions?query=workflow%3ABuild)
+  [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-udp.svg)](https://github.com/ballerina-platform/module-ballerina-udp/commits/master)
+  [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/udp.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fsocket)
+  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+The UDP library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+
+This module provides an implementation for sending/receiving messages to/from another application process (local or remote) for connectionless protocols. 
+
+For more information on the operations supported go to [The UDP Module](https://ballerina.io/swan-lake/learn/api-docs/ballerina/udp/).
+
+For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/swan-lake/learn/by-example/udp-socket-client.html).
```

Comment:
Please make sure we add the the BBE (Ballerina by Example) later with this name.

Reply:
BBE removed from README